src/main/resources/schemas/nhinc/common/NhincCommon.xsd

```diff
@@ -50,6 +50,7 @@
             <xsd:element name="samlAuthzDecisionStatement" type="tns:SamlAuthzDecisionStatementType" minOccurs="0"/>
             <xsd:element name="samlSignature" type="tns:SamlSignatureType" minOccurs="0"/>
             <xsd:element name="samlIssuer" type="tns:SamlIssuerType" minOccurs="0"/>
+            <xsd:element name="samlSubjects" type="tns:SamlSubjectsType" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
Rename to be more descriptive:  samlSubjectConfirmations and SamlSubjectConfirmationType.